notebooks/case1_bike_count.md

```diff
@@ -292,9 +296,7 @@ The count of the possible intervals is of interest:
 df.index.to_series().diff().value_counts()
 ```
 
-There are a few records that are not exactly 15min. But given it are only a few ones, we will ignore this for the current case study and just keep them for this explorative study.
-
-Bonus question: do you know where the values of `-1 days +23:15:01` and `01:15:00` are coming from?
+There are a few records that are not 5min. Ddo you know where the values of `0 days 01:05:00` and the duplicate entries with `0 days 00:00:00` are coming from?
```

Comment:
Note to self (for when this question comes): it's a bit strange that there is still this error, because we actually have the data as timezone aware (and now after processing in UTC), so in theory there should not _need_ to be this gap (if it was in local time without timezone information, they yes, you get such `diff()` values). 

Looking at the original data during the DST, it's simply "wrong" in the data:

![image](https://github.com/jorisvandenbossche/course-python-data/assets/1020496/aae09df3-5464-4b66-ba11-fdb069daf2e3)

So it goes from "2022-10-30T01:55:00+02:00" to "2022-10-30T02:00:00+01:00". While in a correct time series, it should go from "2022-10-30 02:55:00+02:00" to  "2022-10-30T02:00:00+01:00", so there is simply an hour missing in the data.

For the DST in spring, similarly there is an error in the dataset, where the hour what should normally be a gap in local wall time, the data is actually present, giving duplicate values for 1 hour:

![image](https://github.com/jorisvandenbossche/course-python-data/assets/1020496/50af8f3f-a17e-4e4e-b0c4-0c0a301dd246)

Those wrong "non-existing" local times always have zero counts, so shouldn't have an influence on the result.